From 8f0e24c4f1075367c14b5f52ade96a93845e9882 Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa@fedoraproject.org>
Date: Fri, 5 Dec 2025 16:25:47 -0500
Subject: [PATCH] src/qml: LandingComponent: Use the Fedora wallpaper

This ensures that Plasma Setup starts up with the correct wallpaper
branding.
---
 src/qml/LandingComponent.qml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/qml/LandingComponent.qml b/src/qml/LandingComponent.qml
index b922c27..dfde9c2 100644
--- a/src/qml/LandingComponent.qml
+++ b/src/qml/LandingComponent.qml
@@ -44,9 +44,9 @@ Item {
 
         source: {
-            const imgFile = isLandscape ? '5120x2880.png' : '1080x1920.png';
-            const lightWallpaperFolder = 'wallpapers/Next/contents/images/';
-            const darkWallpaperFolder = 'wallpapers/Next/contents/images_dark/';
-
-            const wallpaperUrl = StandardPaths.locate(
-                StandardPaths.GenericDataLocation,
-                (Prepare.PrepareUtil.usingDarkTheme ? darkWallpaperFolder : lightWallpaperFolder) + imgFile
-            );
-            if (!wallpaperUrl) {
-                return StandardPaths.locate(StandardPaths.GenericDataLocation, lightWallpaperFolder + imgFile);
-            }
-            return wallpaperUrl;

+            const imgFileDark = '4725x2658.jpg';
+            const imgFileLight = '5755x3369.jpg';
+            const lightWallpaperFolder = 'wallpapers/Next/contents/images/';
+            const darkWallpaperFolder = 'wallpapers/Next/contents/images_dark/';
+
+            const wallpaperUrl = StandardPaths.locate(
+                StandardPaths.GenericDataLocation,
+                (Prepare.PrepareUtil.usingDarkTheme ? darkWallpaperFolder : lightWallpaperFolder) + (Prepare.PrepareUtil.usingDarkTheme ? imgFileDark : imgFileLight)
+            );
+
+            if (!wallpaperUrl) {
+                return StandardPaths.locate(StandardPaths.GenericDataLocation, lightWallpaperFolder + imgFileLight);
+            }
+            return wallpaperUrl;
-- 
2.52.0

